swagger.yaml

```diff
@@ -196,6 +196,44 @@ paths:
                     - $ref: '#/components/schemas/SearchContactsResult'
         '401':
           $ref: '#/components/responses/UnauthorizedError'
+  /cases/search:
```

Comment:
Is it your plan that when we search, we send two separate requests -- one for `/cases/search` and one for `/contacts/search`?  What are the reasons for doing this instead of one `/search` endpoint?

Reply:
A single search endpoint would have too much logic/complexity into it, making it harder to maintain, test, and even to use it. Handling different paginations in a single endpoint for `cases` and `contacts`, for example, could be tricky.